### PR TITLE
Documentation for settings to disallow concurrent backup/restore

### PR DESCRIPTION
**Pull request by SmitaRKulkarni**
Changelog category (leave one):
- Documentation (changelog entry is not required)

Added documentation for new settings to disallow concurrent backup/restore.
Issue : #43891
PRs : #45072 , #45982

cc @vitlibar

### CODE REVIEW

**Review by DanRoscigno — CHANGES_REQUESTED** [review]
If I got it wrong, then please switch from false to true.

File: docs/en/operations/backup.md [hunk 1]
```diff
@@ -383,3 +383,19 @@ Data can be restored from backup using the `ALTER TABLE ... ATTACH PARTITION ...
...
+<clickhouse>
+    <backups>
+        <allow_concurrent_backups>false</allow_concurrent_backups>
+        <allow_concurrent_restores>false</allow_concurrent_restores>
+    </backups>
+</clickhouse>
+```
+
+The default value for both is true, so by default concurrent backup/restores are allowed.
+When this setting is used on a cluster, only 1 backup/restore is allowed to run on a cluster at a time.
```
[the remark this answers is not in the document]
Comment (SmitaRKulkarni) [reply docs/en/operations/backup.md:401]: it should be true, will update

[the remark this answers is not in the document]
Comment (SmitaRKulkarni) [reply docs/en/operations/backup.md]: addressed it

**Review by DanRoscigno — APPROVED** [review]
LGTM!

**Pull request merged**